packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -150,6 +152,27 @@ export class Meter implements types.Meter {
     this._exporters.push(exporter);
   }
 
+  /**
+   * Provide a pre-computed re-useable LabelSet by
+   * converting the unordered labels into a canonicalized
+   * set of lables with an unique identifier, useful for pre-aggregation.
+   * @param labels user provided unordered Labels.
+   */
+  static labels(labels: types.Labels): types.LabelSet {
+    const keys = Object.keys(labels).sort();
+    const identifier = keys.reduce((result, key) => {
+      if (result.length > 2) {
+        result += ',';
+      }
+      return (result += key + ':' + labels[key]);
+    }, '|#');
+    const sortedLabels: types.Labels = {};
```

Comment:
if `sortedLabels` is a map object why can't you use `labels` ?
or if you want to create a new object then 
`const sortedLabels = Object.assign({}, labels);`